### PR TITLE
DFPL-1574 Remove document Journey (Updated manage document event)

### PR DESCRIPTION
**Pull request by so99y**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DFPL-1574

### Change description ###
- Remove document Journey (Updated manage document event)
- Enable "Change Folder" option

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: service/src/main/java/uk/gov/hmcts/reform/fpl/testingsupport/controllers/TestingSupportController.java [hunk 1]
```diff
@@ -168,6 +169,32 @@ public NotificationList getEmails(@PathVariable("caseId") String caseId) {
...
+            .caseAssignedUserRoles(List.of(CaseAssignedUserRoleWithOrganisation.builder()
+                .caseDataId(caseId.toString())
+                .userId(userId)
+                .caseRole(role)
+                .build()))
+            .build();
+
+        caseAccess.removeCaseUserRoles(userToken, authToken.generate(), accessRequest);
+
+        log.info("Role {} revoked from user {} to case {}", role, email, caseId);
```
Comment (chak-shing-lo-justice) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/testingsupport/controllers/TestingSupportController.java:195]: Why do we need this for document removal? Should we separate it to another PR / ticket?

File: e2e/helpers/api_helper.js [hunk 2]
```diff
@@ -109,6 +109,14 @@ const grantCaseAccess = async (caseId, user, role) => {
...
 };
 
+const revokeCaseAccess = async (caseId, user, role) => {
+  const authToken = await getAuthToken();
+  const url = `${config.fplServiceUrl}/testing-support/case/${caseId}/revoke-access`;
+  const headers = getHeaders(authToken);
+  const data = { email: user.email, password: user.password, role: role };
+  return await post(url, data, headers);
+};
+
```
Comment (chak-shing-lo-justice) [inline e2e/helpers/api_helper.js:119]: Why do we need this for document removal? Should we separate it to another PR / ticket?

**Review by chak-shing-lo-justice — COMMENTED** [review]
LGTM, just some questtion

File: ccd-definition/AuthorisationCaseField/CareSupervision/judiciary.json [hunk 3]
```diff
@@ -5569,196 +5576,196 @@
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseFieldID": "componentLauncher",
     "UserRole": "caseworker-publiclaw-judiciary",
-    "CRUD": "R"
+    "CRUD": "CRU"
```
[the remark this answers is not in the document]
Comment (so99y) [reply ccd-definition/AuthorisationCaseField/CareSupervision/judiciary.json:5579]: It enables the judges to be able to use the "Change Folder" option.

File: ccd-definition/AuthorisationCaseField/CareSupervision/magistrate.json [hunk 4]
```diff
@@ -2503,196 +2510,196 @@
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseFieldID": "componentLauncher",
     "UserRole": "caseworker-publiclaw-magistrate",
-    "CRUD": "R"
+    "CRUD": "CRU"
```
[the remark this answers is not in the document]
Comment (so99y) [reply ccd-definition/AuthorisationCaseField/CareSupervision/magistrate.json:2513]: It enables the magistrates to be able to use the "Change Folder" option.

File: ccd-definition/AuthorisationCaseField/CareSupervision/gatekeeper.json [hunk 5]
```diff
@@ -4925,7 +4932,7 @@
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseFieldID": "componentLauncher",
     "UserRole": "caseworker-publiclaw-gatekeeper",
-    "CRUD": "R"
+    "CRUD": "CRU"
```
[the remark this answers is not in the document]
Comment (so99y) [reply ccd-definition/AuthorisationCaseField/CareSupervision/gatekeeper.json:4935]: same as above. HMTCS staff are able to use the "Change Folder" option in the CFV component.

File: ccd-definition/AuthorisationComplexType/caseDocuments/courtBundleRemoved.json [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "caseworker-publiclaw-courtadmin",
+          "caseworker-publiclaw-gatekeeper",
+          "caseworker-publiclaw-judiciary",
+          "caseworker-publiclaw-magistrate",
+          "caseworker-publiclaw-superuser"
+        ],
+        "CRUD": "U"
```
[the remark this answers is not in the document]
Comment (so99y) [reply ccd-definition/AuthorisationComplexType/caseDocuments/courtBundleRemoved.json:16]: Good spot! Looks like the whole file can be removed. Let me try to remove it and see if any errors.

File: ccd-definition/AuthorisationCaseField/CareSupervision/court-admin.json [hunk 7]
```diff
@@ -5891,7 +5898,7 @@
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseFieldID": "componentLauncher",
     "UserRole": "caseworker-publiclaw-courtadmin",
-    "CRUD": "R"
+    "CRUD": "CRU"
```
[the remark this answers is not in the document]
Comment (so99y) [reply ccd-definition/AuthorisationCaseField/CareSupervision/court-admin.json:5901]: same as above. HMTCS staff are able to use the "Change Folder" option in the CFV component.

File: e2e/helpers/api_helper.js (hunk 2 above)
Comment (so99y) [reply e2e/helpers/api_helper.js:119]: Yes, can extract it to another ticket.

**Review by chak-shing-lo-justice — APPROVED** [review]
lgtm

**Review by jamesrferguson1 — COMMENTED** [review]
Adding manage docs requirement label
